test/_initialization/initialize.test.ts

```diff
@@ -1,46 +1,29 @@
 import { ApiPromise, Keyring } from "@polkadot/api";
 import { KeyringPair } from "@polkadot/keyring/types";
 import * as bitcoinjs from "bitcoinjs-lib";
-import Big from "big.js";
+import { assert } from "chai";
 import BN from "bn.js";
 
-import { 
```

Comment:
Do you think it would make sense to expose the test initialization as part of the lib? I guess this is mostly a duplicate of the code you have in the lib anyway. And if anyone would want to build something with JS based on the lib, they might want to have a similar setup. Definitely for a different PR. Let me know if you think that makes sense though.

Reply:
That's a great idea!